components/button-cards/solution-card/solution-card.hbs

```diff
@@ -0,0 +1,424 @@
+{{!-- <h1>{{general.cardDescription}}</h1> --}}
+<h1>Solution cards</h1>
+<div class="container">
+    <div class="row card-row">
+        <div class="col-12 col-md-4">
+            <div class="msds-solution-card msds-solution-card--certified">
+                <div class="msds-solution-card__header">
+                    <div class="msds-solution-card__header-logo-container">
+                    
+                        <img src=" {{ path '/src/demo-img/visionlabs.jpg' }}" alt="solution card image"
+                            class="msds-solution-card__header-logo-container-logo">
+                    </div>
+                    <div class="msds-solution-card__header-tag-container">
+                        <div class="msds-tag msds-bg-clear-blue msds-text-white">
+                            <div class="msds-icon msds-reliable"></div>
+                            <div class="msds-tag-text msds-text-header-6">Solution</div>
+                        </div>
+                    </div>
+                    <h3 class="msds-solution-card__header-headline msds-text-header-4">
+                        VisionLabs Face Recognition Plug-in
+                    </h3>
+                </div>
+                <div class="msds-solution-card__body">
+                    <h3 class="msds-solution-card__body-headline">
+                        By xPrime
+                    </h3>
+                    <p class="msds-solution-card__body-description ">
+                        Prevent Vandalism | Prevent Vandalism | Control access to restricted areas 
+                    </p>
+                </div>
+                <div class="msds-solution-card__footer">
+                    <div class="msds-solution-card__footer-certification">
+                        <div class="icon-container ">
```

Comment:
why doesn't it belong to solution-card component? so it will be msds-solution-card__footer-icon-container? It's against bem and everyone can specify icon-container class what can mess up your code (or it's even already created since the name sounds really generic). Propability that someone can specify "msds-solution-card__footer-icon-container" class is close to 0%.  Do your code bug-proof man!

Edit:
It can be that this this class is not use at all since i can not find definition in scss file